Geometry2d/src/Arc.cpp

```diff
@@ -2,7 +2,7 @@
 // Created by matt on 7/19/15.
 //
 
-#include "Arc.hpp"
+#include <Geometry2d/Arc.hpp>
```

Comment:
For things in the same package, at least, didn't we agree on quotes instead of angle brackets?

Reply:
Leaving a todo linked to #1483.